core/src/main/scala/kafka/controller/PartitionStateMachine.scala

```diff
@@ -245,11 +247,11 @@ class PartitionStateMachine(controller: KafkaController) extends Logging {
       controllerContext.partitionLeadershipInfo.get(topicPartition) match {
         case Some(currentLeaderIsrAndEpoch) =>
           // else, check if the leader for partition is alive. If yes, it is in Online state, else it is in Offline state
-          controllerContext.liveBrokerIds.contains(currentLeaderIsrAndEpoch.leaderAndIsr.leader) match {
-            case true => // leader is alive
-              partitionState.put(topicPartition, OnlinePartition)
-            case false =>
-              partitionState.put(topicPartition, OfflinePartition)
+          if(controllerContext.liveBrokerIds.contains(currentLeaderIsrAndEpoch.leaderAndIsr.leader)) {
+            // leader is alive
+            partitionState.put(topicPartition, OnlinePartition)
+          }else{
```

Comment:
This is not following our formatting guidelines, there should be a space after `if` and a space before and after `else`.